### DISCUSSION TITLE
Unable to use Google Sign in

### DISCUSSION THREAD

**Discussion by kartikwatwani** [requarks/wiki#6636] [Help / Questions]
I followed the steps mentioned in the docs to activate Google sign in for wiki.js but during the redirect after I give consent I am receiving ERR_CONNECTION_FAILED in the browser. The link that shows this error looks similar to this https://mydomain.com/login/long-code/callback?code=some-code&scope=email+profile+https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fuserinfo.profile+https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fuserinfo.email+openid&authuser=0&hd=anotherDomain.com&prompt=consent#

Can you guide why this is occuring? Also, how can I resolve this?

PS: Mydomain.com is different than anotherDomain.com because I am signing in using different domain email than the domain on which my wiki.js is hosted. But this is intentional, as I wish to login using another domain.

**Comment by NGPixel** [comment]
Check your DNS, network and callback URL configuration. Either you have something misconfigured on Google side or something like a reverse proxy is blocking your requests.
